### ISSUE TITLE
Add file, add folder do not work.

### ISSUE THREAD

**Issue by pruge** [icetee/remote-ftp#52]
remote-ftp@0.6.0

when  Add file, add folder

do not work 
    atom 0.183, 0.184 
work 
    atom 0.182

**Comment by dantman** [comment]
I can confirm this as well. When used from the remote-ftp panel the prompt is empty where it would usually contain the base folder's name in it and typing does not show any text.

However Atom does seem to understand that text is being typed and whatever you do type is accepted and hitting enter does create a file of that name.

So it seems like there is something wrong with the display.

Atom 0.187.0
Theme: One Dark
Syntax Theme: Base16 Tomorrow Dark

**Comment by dantman** [comment]
This appears to be require-ftp's DOM:

``` html
<atom-panel class="modal overlay from-top">
    <div class="tree-view-dialog overlay from-top" callattachhooks="true">
        <label class="icon icon-file-directory-create">Enter the path for the new folder.</label>
        <atom-text-editor class="editor mini" tabindex="-1" callattachhooks="true" data-grammar="text plain null-grammar" mini="" data-encoding="utf8">
            <div class="underlayer">
            </div>
            <div class="overlayer">
            </div>
        </atom-text-editor>
        <div class="error-message">
        </div>
    </div>
</atom-panel>
```

This appears to be the DOM of the standard new file dialog (which works):

``` html
<atom-panel class="modal overlay from-top">
    <div class="tree-view-dialog" is="space-pen-div">
        <label class="icon icon-file-add">Enter the path for the new file.</label>
        <atom-text-editor class="editor mini" tabindex="-1" callattachhooks="true" mini="" data-grammar="text plain null-grammar" data-encoding="utf8">
        </atom-text-editor>
        <div class="error-message">
        </div>
    </div>
</atom-panel>
```

**Comment by grenierdev** [comment]
Fixed in eb46f06911c402703e945b11c26fe21617984d9b
